CliFx/Domain/CommandArgumentSchema.cs

```diff
@@ -52,17 +83,17 @@ protected CommandArgumentSchema(PropertyInfo property, string? description)
                 // String-constructable
                 var stringConstructor = GetStringConstructor(targetType);
                 if (stringConstructor != null)
-                    return stringConstructor.Invoke(new object[] {value!});
+                    return stringConstructor.Invoke(new object[] { value! });
```

Comment:
Whitespace changes 😱

Reply:
Ooops! I think Roslynator is trolling us whenever I fix the whitespace formatting. Fixed :)